.github/workflows/pythonapp.yml

```diff
@@ -36,19 +36,19 @@ jobs:
         # The GitHub editor is 127 chars wide
         flake8 . --count  --max-complexity=10 --max-line-length=127 --statistics
     # PyTorch for Mac has different pip syntax wrt Win and Linux.
-    # We want to test torch 1.5.x (in this case on python3.6) and torch 1.7.x.
+    # We want to test torch 1.6.0 (in this case on python3.7) and torch 1.10.x.
```

Comment:
Why do we do this? I don't remember. Is this because for 3.5 we were only able to run PyTorch 1.5.1? If true maybe we can just test on 1.10.x and remove all these conditions.

Reply:
Now we are testing python 3.7 on torch 1.6, and 1.10 on all other pythons.

I think we did it just to test older torch versions?

You are right, we could just 1.10 on ALL of these...  But I guess that helped us test our `LooseVersion` bounds, so maybe we leave it?